src/canonmn_int.cpp

```diff
@@ -2851,6 +2850,33 @@ std::ostream& printCsLensFFFF(std::ostream& os, const Value& value, const ExifDa
   return EXV_PRINT_TAG(canonCsLensType)(os, value, metadata);
 }
 
+/**
+ * @brief convert string to float w/o considering locale
+ *
+ * Using std:stof to convert strings to float takes into account the locale
+ * and thus leads to wrong results when converting e.g. "5.6" with a DE locale
+ * which expects "," as decimal instead of ".". See GitHub issue #2746
+ *
+ * Use std::from_chars once that's properly supported by compilers.
```

Comment:
Out of curiosity, which one doesn't?

Reply:
https://en.cppreference.com/w/cpp/compiler_support/17#C.2B.2B17_library_features